src/app/wizard/wizard.component.less

```diff
@@ -13,9 +13,10 @@
   position: relative;
 }
 .wizard-pf-footer {
-  .btn-cancel {
-    &.wizard-pf-cancel-no-back {
-      margin-right: 0;
```

Comment:
Not sure what this was intended to do, but thanks for finding it. Obviously, it does not work properly.

This was copied over from Angular Patternfly.